PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## What
+_Describe what this PR is doing._ 
+
+## Why ?
+_Justification for the PR. If there is existing issue/bug please reference. For
+bug fixes why and what can be merged in a single item._
+
+## How ?
+_It is optional but for complex PRs please provide information about the design,
+architecture, approach, etc.
```

Comment:
## How it was tested

## todo/limitations (if applicable)

## Performance before/after (if applicable)

maybe worth adding "checklist" as well, smth like:

## checklist

- [] for major feature: add wiki page with usage example and description 
- []

Reply:
WDYT

Reply:
My goal was to keep it as simple as possible. Describe in layman's terms what you are doing.
Later on we can extend this. As for now I would love to see clear description why we introduce a change.